static/js/StaticPages.jsx

```diff
@@ -3140,19 +3140,19 @@ const JobsPage = memo(() => {
 const DevBox = () => {
     return (
       <div className='productsDevBox'>
-        <p className='productsDevHeader'>
+        <div className='productsDevHeader'>
```

Comment:
Please note that we have (some small) styling for [`.productsDevBox p`](https://github.com/Sefaria/Sefaria-Project/blob/0a4c5722b7aae9934c9c8d5fa774c55e7675a397/static/css/s2.css#L13547), so just double check the visuals before and after, and change as necessary.

Reply:
Will do!

Reply:
Was extra, thanks for catching. Pushing a fix that deletes it.